draft/spec/index.html

```diff
@@ -768,24 +767,9 @@ <h2>Storage Hierarchies</h2>
           <li>Storage hierarchies MUST be terminated by OCFL Object Roots</li>
           <li>Storage hierarchies within the same OCFL Storage Root SHOULD use just one layout pattern</li>
           <li>Storage hierarchies within the same OCFL Storage Root SHOULD consistently use either a directory
-            hierarchy of OCFL Objects, top-level OCFL Objects, or OCFL Object ZIPs
+            hierarchy of OCFL Objects or top-level OCFL Objects
```

Comment:
missing a period.

Reply:
there aren't periods on the other bullets... all or none!

Reply:
I am not a fan of periods in lists.

Reply:
comedy

Reply:
😁

Reply:
I could have sworn I saw periods in the list.  oh well.  FINE!